Makefile

```diff
@@ -326,10 +326,8 @@ gcs-publish-ci: gcs-upload
 	gsutil -h "Cache-Control:private, max-age=0, no-transform" cp ${UPLOAD}/${LATEST_FILE} ${GCS_LOCATION}
 
 .PHONY: gen-cli-docs
-gen-cli-docs: ${KOPS} # Regenerate CLI docs
```

Comment:
I don't think we're (quite) ready to remove the non-bazel form (?)

Reply:
That was unintentional

Reply:
I am not getting an error running that target with go 1.10.  Arg will debug more